Sources/Views/Cells/AudioMessageCell.swift

```diff
@@ -47,25 +47,51 @@ open class AudioMessageCell: MessageContentCell {
         return durationLabel
     }()
 
+    private lazy var activityIndicatorView: UIActivityIndicatorView = {
+        let activityIndicatorView = UIActivityIndicatorView(style: .gray)
+        activityIndicatorView.hidesWhenStopped = true
+        activityIndicatorView.isHidden = true
+        return activityIndicatorView
+    }()
+
     public lazy var progressView: UIProgressView = {
         let progressView = UIProgressView(progressViewStyle: .default)
         progressView.progress = 0.0
         return progressView
     }()
 
+    public var isLoadingIndicatorVisible: Bool {
```

Comment:
The naming of this variable doesn't line up with the value.
ex: Given `activityIndicatorView.isHidden = true`, `isLoadingIndicatorVisible` would return `true`, but that would indicate that `activityIndicatorView.isHidden` is false 

If we make `activityIndicatorView` `public`, I think we could actually move this to a function. Like

`startLoadingAnimation()` and `stopLoadingAnimation()` as in its current form I think its less obvious to expect that the `playButton` becomes hidden

Reply:
I generally like my approach, but I removed this since the `activityIndicatorView` is now public. Users can do whatever they want with it and I moved this approach into an extension in my codebase.